datasets/case01/queries/1002_coneMagColor_noalias.sql

```diff
@@ -1,7 +1,7 @@
 -- Cone-magnitude-color search
 -- See http://dev.lsstcorp.org/trac/wiki/dbQuery003
 
--- pragma noheader
+-- pragma noheader sortresult
```

Comment:
Oh this works? I expected we'd be doing each pragma in separate line. I assume you tested it :)